corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -194,3 +194,8 @@ def get_data_schema(request, domain, app_id=None, form_unique_id=None):
     if "pretty" in request.GET:
         kw["indent"] = 2
     return HttpResponse(json.dumps(data, **kw))
+
+
+@require_GET
```

Comment:
the only thing I'm not sure about in there is [this line](https://github.com/dimagi/commcare-hq/blob/master/corehq/middleware.py#L123), which checks for `request.domain`. Is that set on URLs (like http://www.commcarehq.org/ping/) that do not include the domain? Like is `request.domain` loaded from a cookie or the session or something like that for those URLs? If not, I may need to move the `ping` url into the domain namespace.